src/main/java/frc/robot/config/Config.java

```diff
@@ -68,13 +70,23 @@ private Config() {
     public static boolean INVERT_FORWARD = robotSpecific(true, true, true);
     public static boolean INVERT_SIDE= robotSpecific(false, false, false);
 
+    public static boolean INVERT_ARM_TALON = robotSpecific(false, false, false);
+
+    public static int ARM_ALLOWABLE_CLOSED = 4096;
+
     // Timeouts for sending CAN bus commands
     public static final int CAN_TIMEOUT_SHORT = 10;
     public static final int CAN_TIMEOUT_LONG = 100;
 
+    // PIDF values for the arm
+    public static double ARM_P = robotSpecific(0.2);
```

Comment:
Nitpick: I understand what these mean as I see them together. For clarity you may want to consider ARM_PID_P_VALUE, etc.